tests/test_e2e_80_pathfinder.py

```diff
@@ -0,0 +1,160 @@
+    net = None
+
+    def setup_method(self, method):
+        """
+        It is called at the beginning of every class method execution
+        """
+        # Start the controller setting an environment in
+        # which all elements are disabled in a clean setting
+        self.net.start_controller(clean_config=True, enable_all=True)
+        self.net.wait_switches_connect()
+        time.sleep(10)
+
+    @classmethod
+    def setup_class(cls):
+        cls.net = NetworkTest(CONTROLLER, topo_name="multi")
+        cls.net.start()
+        cls.net.restart_kytos_clean()
+        cls.net.wait_switches_connect()
+        time.sleep(5)
+
+    @classmethod
+    def teardown_class(cls):
+        cls.net.stop()
+
+    def restart(self, _clean_config=False, _enable_all=False):
+
+        # Start the controller setting an environment in which the setting is
+        # preserved (persistence) and avoid the default enabling of all elements
+        self.net.start_controller(clean_config=_clean_config, enable_all=_enable_all)
+        self.net.wait_switches_connect()
+
+        # Wait a few seconds to kytos execute LLDP
+        time.sleep(10)
+    
+    def add_topology_metadata(self):
+        """Add topology metadata."""
+        links_metadata = {
+            "74bbc9527a0e309a86c95744042bcf9e3beb52955c942cac5fc735b1cf986f7f": {
+                "link_name": "s1-eth3-s6-eth3",
+                "ownership": "red",
+                "bandwidth": "10",
+                "delay" : 100,
+                "priority": 120,
+            },
+            "cf0f4071be426b3f745027f5d22bc61f8312ae86293c9b28e7e66015607a9260": {
+                "link_name": "s1-eth2-s2-eth2",
+                "ownership": "blue",
+                "bandwidth": "100",
+                "delay" : 10,
+                "priority": 5,                
+            },
+            "adda3859b963110d584bf6ec3ac85ddea80276001e37edc1c420463a34c80c9e": {
+                "link_name": "s2-eth4-s6-eth4",
+                "ownership": "blue",
+                "bandwidth": "100",
+                "delay" : 10,
+                "priority": 5,                
+            },
+            
+        }
+
+        for link_id, metadata in links_metadata.items():
+            api_url = f"{KYTOS_API}/topology/v3/links/{link_id}/metadata"
+            response = requests.post(
+                api_url,
+                data=json.dumps(metadata),
+                headers={"Content-type": "application/json"},
+            )
+            assert response.status_code == 201, response.text
+        return links_metadata        
+
+    @pytest.mark.parametrize("undesired_link, expected", 
+                            [([], 1),(["74bbc9527a0e309a86c95744042bcf9e3beb52955c942cac5fc735b1cf986f7f"], 8)])
+    def test_10_undesired_link_and_max_path(self, undesired_link, expected):
+        """Tests fastest path from switch 1 to 6 then blocks it"""
+        api_url = KYTOS_API + '/pathfinder/v3/'
+        post_body = {
```

Comment:
The `post_body` indentation is a bit non standard, we don't have linters integrated on this repo yet, but I'll recommend that you run [`black tests/test_e2e_80_pathfinder.py`](https://github.com/psf/black).